### DISCUSSION TITLE
Default empty results

### DISCUSSION THREAD

**Discussion by JohnMoor** [microsoft-search/pnp-modern-search#1463] [Q&A]
I'm trying PNP Search box and Results and notice that the Search results show content already by default without anything text in the search box. I connected the search box to the search results. 

Is there any way to configure the search result to be empty by default unless the search box is used?

Apologies if I'm missing something obvious here.

**Comment by FranckyC** [comment]
Hi @JohnMoor, to get empty results at page load, make sure:
- In the Search Results Web Part, there is no default input query
- In the Search Results Web Part, the query template is only `{searchTerms}`
